secure_fw/spm/cmsis_func/spm_func.c

```diff
@@ -50,9 +50,16 @@ struct iovec_params_t {
 #error Multi core is not supported by Function mode
 #endif
 
-REGION_DECLARE_T(Image$$, TFM_SECURE_STACK, $$ZI$$Base, uint32_t);
-REGION_DECLARE_T(Image$$, TFM_SECURE_STACK, $$ZI$$Limit, struct iovec_args_t *);
-REGION_DECLARE_T(Image$$, ER_INITIAL_PSP_SEAL, $$ZI$$Base, uint32_t *);
+REGION_DECLARE(Image$$, TFM_SECURE_STACK, $$ZI$$Base);
+REGION_DECLARE(Image$$, TFM_SECURE_STACK, $$ZI$$Limit);
+REGION_DECLARE_T(Image$$, ER_INITIAL_PSP_SEAL, $$ZI$$Base, uint32_t)[2];
+
+static inline uint32_t *tfm_secure_stack_iovec_get(void)
+{
+    uint32_t secure_stack_limit = (uint32_t)&REGION_NAME(Image$$, TFM_SECURE_STACK, $$ZI$$Limit);
```

Comment:
technically type should be uintptr_t

Reply:
Yeah, but all uses of REGION_NAME has uint32_t, I think I will leave it even though I agree with you.